sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -596,6 +659,56 @@ void validateRecognizedResult(List<RecognizedForm> actualFormList, boolean inclu
         }
     }
 
+    // Business cards
+    void validateBusinessCardDataFields(Map<String, FormField> actualRecognizedBusinessCardFields, boolean includeFieldElements) {
+        final AnalyzeResult analyzeResult = getAnalyzeRawResponse().getAnalyzeResult();
+        List<ReadResult> readResults = analyzeResult.getReadResults();
+        DocumentResult documentResult = analyzeResult.getDocumentResults().get(0);
+        Map<String, FieldValue> expectedReceiptFields = documentResult.getFields();
+
+        validateFieldValueTransforms(expectedReceiptFields.get(CONTACT_NAMES),
```

Comment:
In response to [this](https://github.com/Azure/azure-sdk-for-java/pull/15829/files/4ca941908e952d64df5b77759742a12796075610#r504325546)

Could do something like, 
```java
businessCardsFields.forEach(eachField -> 
     validateFieldValueTransforms(expectedReceiptFields.get(eachField), actualRecognizedBusinessCardFields.get(eachField), readResults, includeFieldElements))
```